README.md

```diff
@@ -87,6 +87,20 @@ vars:
     product_variant_pass_through_columns: []
 ```
 
+### Adding Metafields
+In [May 2021](https://fivetran.com/docs/applications/shopify/changelog#may2021) the Shopify connector included support for the [metafield resource](https://shopify.dev/api/admin-rest/2023-01/resources/metafield). If you would like to take advantage of these metafields, this package offers corresponding mapping models which append these metafields to the respective source object for the following tables: collection, customer, order, product_image, product, product_variant, shop. If enabled, these models will materialize as `shopify__[object]_metafields` for each respective supported object. To enable these metafield mapping models, you may use the following configurations within your `dbt_project.yml`.
```

Comment:
should we call out how customers can use the metafield macro for tables not included in the package?

Reply:
also should we make it clearer that the output models will be the same as the staging ones, but with the metafields pivoted out into columns

Reply:
I had considered the macro definition but noted above that it became quite lengthy and added a lot of noise. I was wondering if we should leave these metafields just in the package for now. We can then see what the reception to them is. 

I also was getting worried about handling issues coming into this package that are around the custom implementation of the macro into customers projects. I think we should wait to ensure this works at scale with our package users before providing more detail in this readme.

I do think it would be relevant to add that these will be the same output to the staging models.

Reply:
Updated the wording to include a note about the record count should match the staging model.

Reply:
ah missed your other comment -- yep completely agree and the note sounds good